src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -188,6 +188,19 @@ internal int CurrentPasteOffset
         /// </summary>
         private bool workspaceLoaded;
 
+        /// <summary>
+        /// This event is to notify that the dummy nodes have been resolved. 
```

Comment:
what about: `This event is raised after the workspace tries to resolve dummyNodes - for example after a new package or library is loaded.`